api/health.go

```diff
@@ -48,17 +48,17 @@ func (s *SystemdUnits) GetUnitsProperties(cfg *Config, tools DCOSHelper) (health
 	excludeUnits := []string{"dcos-setup.service", "dcos-link-env.service", "dcos-download.service"}
 	for _, unit := range foundUnits {
 		if isInList(unit, excludeUnits) {
-			logrus.Debugf("Skipping blacklisted systemd unit %s", unit)
+			logrus.Debugf("Skipping blacklisted systemd Unit %s", unit)
```

Comment:
Minor nit: do we mean to be capitalizing "unit" in string literals here and elsewhere?